reactive/openstack_service_checks.py

```diff
@@ -198,15 +199,32 @@ def render_config():
     if not creds:
         return
 
-    if helper.charm_config["trusted_ssl_ca"].strip():
-        trusted_ssl_ca = helper.charm_config["trusted_ssl_ca"].strip()
+    trusted_ssl_ca = helper.charm_config["trusted_ssl_ca"].strip()
+    if trusted_ssl_ca:
         hookenv.log("Writing ssl ca cert:{}".format(trusted_ssl_ca))
         cert_content = base64.b64decode(trusted_ssl_ca).decode()
+        certs = cert_content.split("-----END CERTIFICATE-----")
+        certs = [cert.strip() + "\n-----END CERTIFICATE-----\n" for cert in certs if cert.strip()]
+
+        for idx, cert in enumerate(certs):
```

Comment:
I think with this method we'll need to clean up existing certs before writing new ones - if the certificate is updated, the charm could leave stale certs in use (if it went from two certificates bundled to just one for example).

eg. something akin to `rm -f $CERT_DIR/openstack-service-checks*`

Reply:
Thanks! I added removing part.